javascript/require-and-import-mini-lesson.md

```diff
@@ -0,0 +1,351 @@
+# Require and Import Mini Lesson
+
+### Projected Time
+
+Example: 2h15 min
+
+- Lesson: 45 min
+- Guided Practice: 60 min
+- Check for Understanding: 30 min
+
+### Prerequisites
+
+Here are topics that should be understood before this topic:
+
+- [JavaScript 1 - Variables, Strings, Numbers Lesson](/javascript/javascript-1-variables.md)
+- [JavaScript 2 - Arrays, Functions Lesson](/javascript/javascript-2-arrays-functions.md)
+- [JavaScript 3 - Conditionals, Comparisons, Booleans Lesson](/javascript/javascript-3-conditionals.md)
+- [JavaScript 4 - Loops Lesson](/javascript/javascript-4-loops.md)
+- [JavaScript 5 - Switch Statements Lesson](/javascript/javascript-5-switch.md)
+- [JavaScript 6 - Object Literals Lesson](/javascript/javascript-6-object-literals.md)
+- [JavaScript 7 - Object-Oriented Programming](/javascript/javascript-7-oop.md)
```

Comment:
These aren't really prereqs so I would just replace this all with Node: `node-js/node-js.md`

Reply:
fixed in [c89273b](https://github.com/Techtonica/curriculum/pull/1414/commits/c89273b844a10c1f67a1d0886a773f18a1c327e8)